### PR TITLE
Updated RU, UA, and global test lists

### PR DESCRIPTION
**Pull request by agrabeli**
This PR includes updates to the UA, RU, and global test lists as provided by a community member.

### CODE REVIEW

File: lists/ru.csv [hunk 1]
```diff
@@ -1,214 +1,289 @@
...
-http://www.wikia.com/wikia,HOST,Hosting and Blogging Platforms,2014-04-15,citizenlab,
-http://www.wuor.ru,HUMR,Human Rights Issues,2014-04-15,citizenlab,
-http://www.yabloko.ru,POLR,Political Criticism,2014-04-15,citizenlab,
-http://www.yandex.ru,NEWS,News Media,2014-04-15,citizenlab,
-http://xn--80aaifmgl1achx.xn--p1ai,GMB,Gambling,2014-04-15,citizenlab,
-http://yesche.biz,ALDR,Alcohol & Drugs,2014-04-15,citizenlab,
-http://zabormozga.com,ALDR,Alcohol & Drugs,2014-04-15,citizenlab,
-http://zhurnal.lib.ru,MISC,Miscelaneous content,2014-04-15,citizenlab,
+url,category_code,category_description,date_added,source,notes
+http://2ch.hk,PORN,Pornography,2014-04-15,citizenlab,
```
Comment (darkk) [inline lists/ru.csv]: Hmmm, it's strange that this line got no edits, but it sill looks like
```diff
 -http://2ch.hk,PORN,Pornography,2014-04-15,citizenlab, 
 +http://2ch.hk,PORN,Pornography,2014-04-15,citizenlab,
```

Maybe something is wrong with formatting of this patch...

File: lists/ua.csv [hunk 2]
```diff
@@ -1,887 +1,941 @@
...
-http://youtube.com,NEWS,News Media,2014-04-15,citizenlab,
-http://yury108.blogspot.com,NEWS,News Media,2014-04-15,citizenlab,
-http://zabormozga.com,ALDR,Alcohol & Drugs,2014-04-15,citizenlab,
-http://zaukraine.com/blog,NEWS,News Media,2014-04-15,citizenlab,
-http://zhurnal.lib.ru,MISC,Miscelaneous content,2014-04-15,citizenlab,
-http://zn.ua,NEWS,News Media,2014-04-15,citizenlab,
-http://znaesh.net,SRCH,Search Engines,2014-04-15,citizenlab,
+url,category_code,category_description,date_added,source,notes,,,,,,,
+http://24tv.ua,NEWS,News Media,2014-04-15,citizenlab,,,,,,,,
+http://2ch.hk,PORN,Pornography,2014-04-15,citizenlab,,,,,,,,
```
Comment (darkk) [inline lists/ua.csv]: Same story here, but it's clear here -- there are some buggy empty columns in the end of each line.

Heh, forms, databases... Now I clearly see the reason for them ;-)

@hellais look, here is an argument :-)

File: lists/ru.csv (hunk 1 above)
Comment (hellais) [reply lists/ru.csv]: That is probably due to the fact that `^M` != `\n` and some people use windows to edit the `.csv` files.

File: lists/ua.csv (hunk 2 above)
Comment (hellais) [reply lists/ua.csv]: Looks like this is something the lint script should detect

Comment (hellais) [reply lists/ua.csv]: It did detect it actually :P

**Comment by hellais** [comment]
@agrabeli can you fix the issues with the wrong column count?

**Comment by agrabeli** [comment]
I just received new test lists for RU, UA, global -- including these URLs plus new ones. I'll therefore close this PR and open a new one, so that everything can be cleaner.

**Pull request closed without merging**